### PR TITLE
improve log for unknown columns in start.R, reduce redundancies in tutorials 2 and 3

### PR DESCRIPTION
**Pull request by orichters**
First, Renato asked me to be improve the logging of unknown columns found in scenario_config files. Here we go, it now states more precisely what is checked and how to treat eventual false positives that may still occur.

Second, I revised tutorials 2 and 3, putting all the stuff on multiple scenario runs into number 3 and replacing outdated information such as `start_bundle.R`. Therefore, I add the original authors of this tutorial as reviewers.

### CODE REVIEW

File: tutorials/3_RunningBundleOfRuns.md [hunk 1]
```diff
@@ -3,36 +3,56 @@ Running more than one REMIND scenario
...
 
-The first two columns are mandatory and usually placed at the beginning.
+To compare a `scenario_config*.csv` file to the current default configuration, you can run `Rscript -e "remind2::colorScenConf()"` in your remind directory and select the file you are interested in. [`colorScenConf()`](https://github.com/pik-piam/remind2/blob/master/R/colorScenConf.R) produces a file ending with `_colorful.xlsx` in the same directory and provides you with information how to interpret the colors within.
 
-* `title` contains a unique identifier for each run, which must not contain a `.` and must not end with a `_`.
-* `start` is a boolean switch, either 1 if the run is to be executed once run with `start.R`, or 0 if not.
+An important feature of scenario_config files is the possibility to execute runs which build on each other.
+An example is a base run without climate policies and a policy run with carbon taxes.
+Such a "subsequent run" has to wait for the run it builds on to be finished.
```
Comment (LaviniaBaumstark) [inline tutorials/3_RunningBundleOfRuns.md:37]: maybe add here the most common example for subsequent runs where the first time steps until `startyear` are used form the first scenario.

**Review by LaviniaBaumstark — APPROVED** [review]

File: tutorials/3_RunningBundleOfRuns.md [hunk 2]
```diff
@@ -3,36 +3,56 @@ Running more than one REMIND scenario
...
   * If the run with this `title` is set to `start = 0` or does not exist in the `scenario_config_XYZ.csv` file, the function `configure_cfg` of [`start.R`](../start.R) searches in `./output` for folder with that title which contain a `fulldata.gdx` and whose `log.txt` states `REMIND run finished!`, and then picks the one with the latest date and time in the folder name. Appending a `_` to a `path_gdx…` entry, yielding for example `BAU_Nash_`, forces REMIND to take `fulldata.gdx` from earlier `BAU_Nash` runs, but avoids to be counted as a subsequent run. This way, you can quickly switch between the two options.
 
 The image above shows these possibilities used for `path_gdx_ref`. Run `RCP20` has a complete path specified but will not be executed because `start = 0`. `RCP37` provides only the folder name, `RCP26_subsequent` waits for `SSP2-Base` to be finished, and `RCP26_forceoldrun` selects the latest already finished `SSP2-Base` run in the output folder and starts immediately. If you set `start = 0` also for `SSP2-Base`, then `RCP26_subsequent` will also try to find an old run in a folder that looks like `SSP2-Base_YYYY-MM-DD_HH.MM.SS`.
```
Comment (dklein-pik) [inline tutorials/3_RunningBundleOfRuns.md:58]: Could you add which behaviour is expected for `RCP37`? Start immediately if `SSP2-Base_2021-...` finished, otherwise turned into “subsequent runs“, right?

Comment (dklein-pik) [inline tutorials/3_RunningBundleOfRuns.md:58]: Maybe mention, that a subsequent run does not have to have "subsequent" in its name. `RCP26_subsequent` in this example was named this way to make it more explicit that the start.R script will classify it as a subsequent run.

Comment (dklein-pik) [inline tutorials/3_RunningBundleOfRuns.md:58]: In the next paragraph (not part of this PR so far) you mention "the five mandatory columns". Could you list them?

**Review by dklein-pik — COMMENTED** [review]
Great work! Thank you for this very helpful description. It will make the life of newcomers much easier.

File: tutorials/3_RunningBundleOfRuns.md (hunk 2 above)
Comment (orichters) [reply tutorials/3_RunningBundleOfRuns.md:58]: no :). Because since our changes, the `path_gdx…` columns are not mandatory anymore. Thanks for spotting that :)

Comment (orichters) [reply tutorials/3_RunningBundleOfRuns.md:58]: no, as there is no scenario with that long `title`, it looks for a `fulldata.gdx` in this folder name and starts directly. If it doesn't find a file, `start.R` stops with an error message.

**Comment by orichters** [comment]
Thanks for your comments, it helped further improve the tutorials.

Inspired by the remind meeting, I added some more logging to `prepare_and_run.R`: It tells you where it got the gdx files from, and also implements the checklist from the [debugging tutorial](https://gitlab.pik-potsdam.de/REMIND/tutorials/debugging-remind-for-profit-and-fun/-/wikis/Debugging-Tutorial-1) and directly gives you the answer, which will probably make it much easier for starters.

After stating that remind finished, `log.txt` then states:
```
Model summary:
  gams_runtime is 10.2567827701569
  full.gms exists, so the REMIND GAMS code was generated.
  full.log and full.lst exist, so GAMS did run.
  abort.gdx does not exist, which is a file written automatically for some types of errors.
! fulldata.gdx does not exist, so output generation will fail.
! full.log states: *** Status: Compilation error(s)
```

Additionally, I fixed a bug with `o_iterationNumber` in `negishi` and `testOneRegi` as discussed with Lavinia.

File: tutorials/3_RunningBundleOfRuns.md (hunk 2 above)
Comment (orichters) [reply tutorials/3_RunningBundleOfRuns.md]: I did that

File: tutorials/3_RunningBundleOfRuns.md (hunk 1 above)
Comment (orichters) [reply tutorials/3_RunningBundleOfRuns.md]: I did

**Review by LaviniaBaumstark — APPROVED** [review]
nice
